release_notes.md

```diff
@@ -4,3 +4,4 @@
 -->
 
 - Bump protobuf version to v1.5.9-protofile (#1148)
+- Remove preview suffix for Worker.Extensions.Tables package (#1173)
```

Comment:
I think you need to change the release notes inside this extension directory.
https://github.com/Azure/azure-functions-dotnet-worker/blob/main/extensions/Worker.Extensions.Tables/release_notes.md